static/app/views/dashboardsV2/detail.tsx

```diff
@@ -526,7 +533,7 @@ class DashboardDetail extends Component<Props, State> {
                 onAddWidget={this.onAddWidget}
                 onDelete={this.onDelete(dashboard)}
                 dashboardState={dashboardState}
-                widgetCount={dashboard.widgets.length}
+                widgetLimitReached={dashboard.widgets.length >= MAX_WIDGETS}
```

Comment:
I think you can use `this.widgetLimitReached` everywhere you have this since you defined it with the `get` keyword

Reply:
oh oops, forgot to make the change after i added the function, thanks!

Reply:
d95a19b527598d6a47194368e73b97b5e99f2011v